frontend/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -195,21 +195,40 @@ class DatasetInfoTabView extends React.PureComponent<Props> {
     }
     const tracingDescription = this.props.tracing.description || "<no description>";
 
-    return (
-      <div className="flex-overflow">
-        <p>{annotationTypeLabel}</p>
-        <p>
+    let descriptionEditField;
+    if (this.props.tracing.restrictions.allowUpdate) {
+      descriptionEditField = (
+        <span style={{ verticalAlign: "top" }}>
+          Description:
+          <EditableTextLabel
+            value={tracingDescription}
+            onChange={this.setAnnotationDescription}
+            rows={4}
+            markdown
+            label="Annotation Description"
+          />
+        </span>
+      );
+    } else {
+      if (this.props.tracing.description) {
```

Comment:
I would remove the if condition and simply use `<Markdown />` even if tracingDescription equals `<no description>`.